wgsl/index.bs

```diff
@@ -558,6 +656,11 @@ type.
 * type must be `f32`, `i32` or `u32`
 
 ### Read-only Storage Texture Types ### {#texture-ro}
+
+A <dfn noexport>read-only texture</dfn> is an image from which individual texels
```

Comment:
Should this `image` be `texture`? `A read-only texture is a texture from which ...` The switch from texture to image seems confusing